### PR TITLE
8.0 crm claim type dev osval

### PR DESCRIPTION
**Pull request by osvalr**
This module was extracted from vauxoo/rma for vauxoo/crm and called crm_claim_type following suggestions listed https://github.com/OCA/rma/pull/35#issuecomment-128056338

### CODE REVIEW

File: crm_claim_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (moylop260) [inline crm_claim_type/README.rst]: No miss repalce {project_repo} by real data

File: crm_claim_type/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'CRM Claim Types',
+    'category': 'Customer Relationship Management',
+    'summary': 'Claim types for CRM',
+    'author': 'Vauxoo',
```
Comment (moylop260) [inline crm_claim_type/__openerp__.py]: Add license

Comment (moylop260) [inline crm_claim_type/__openerp__.py]: Add to OCA as author too.

File: crm_claim_type/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    'website': 'www.vauxoo.com',
+    'description': """
+    Allows to set and maintain claim types for CRM Claims
+    """,
+    'depends': [
+        'base',
```
Comment (moylop260) [inline crm_claim_type/__openerp__.py]: `base` is redudant

File: crm_claim_type/README.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
+CRM Claim Types
+==============
+
+This module was written to extend crm_claim module features.
+It gives a section to define a set of claim types.
```
Comment (nhomar) [inline crm_claim_type/README.rst]: Add the specific use cas it solves.....

The expresion "This module was written" should not start the documentation of any modules "Iti onvious that it is a module and we see letters there eveybody know it was written jejeje".

Then it should say.

Include a way to clasify claims adding the concept of "type".

IMAGE WITHOUT THIS MODULE:

IMAGE WITH THIS MODULE:

File: crm_claim_type/README.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Installation
+============
+
+To install this module, you just need to select it from availables modules
+
+Configuration
+=============
+
+This modules do not require special configurations.
+It contains a predefined set of claim types.
```
Comment (nhomar) [inline crm_claim_type/README.rst]: Here add:

But if you want add your own types got to there there and there.

File: crm_claim_type/models/crm_claim.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+class CrmClaim(models.Model):
+    """
+        CRM Claim
+    """
+    _inherit = 'crm.claim'
+
+    claim_type = \
+        fields.Many2one('crm.claim.type',
+                        string='Claim Type',
+                        help="Customer: When customer makes a claim to the company.\n"
```
Comment (moylop260) [inline crm_claim_type/models/crm_claim.py:36]: line to loog you can change this line by

``` python
claim_type = fields.Many2one(
    'crm.claim.type',
    string='Claim Type',
    help="Customer: When customer makes a claim to the company.\n"
             "Supplier: When the company makes a claim to supplier.") 
```

File: crm_claim_type/README.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Sales > Configurations > Claims > Claim Types
```
Comment (nhomar) [inline crm_claim_type/README.rst]: This is a configration, not an usage.

Usage is what you can do one time you add a configuration.

File: crm_claim_type/__openerp__.py (hunk 2 above)
Comment (nhomar) [inline crm_claim_type/__openerp__.py]: add OCA please in order to be ready for merge.

Comment (yaniaular) [inline crm_claim_type/__openerp__.py]: @osvalr Like this https://github.com/OCA/rma/pull/32/files#diff-e4f8bc2bd35f91a5974c441fc33cdd7dR75

File: crm_claim_type/models/crm_claim.py (hunk 6 above)
Comment (moylop260) [inline crm_claim_type/models/crm_claim.py:36]: If you use the same name of field and label you can use default and don't send `string` param

File: crm_claim_type/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'CRM Claim Types',
+    'category': 'Customer Relationship Management',
+    'summary': 'Claim types for CRM',
+    'author': 'Vauxoo',
+    'website': 'www.vauxoo.com',
+    'description': """
```
Comment (nhomar) [inline crm_claim_type/__openerp__.py]: IF REAMDE.rst exists this Key should not.

File: crm_claim_type/__init__.py [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
```
Comment (moylop260) [inline crm_claim_type/__init__.py]: License comment

**Comment by osvalr** [comment]
@nhomar @moylop260 your comments were taken in count

**Comment by osvalr** [comment]
@nhomar after this PR takes effect, please make a proposal to integrate it for OCA/crm

File: crm_claim_type/__openerp__.py [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# ##############################################################################
+{
+    'name': 'CRM Claim Types',
+    'category': 'Customer Relationship Management',
+    'summary': 'Claim types for CRM',
```
Comment (moylop260) [inline crm_claim_type/__openerp__.py:28]: Add

``` python
license: 'AGPL-3',
```

**Comment by nhomar** [comment]
NO se si no entendí bien....

El módulo no iba a morir? esperaba un PR con la integración al otro no?

**Comment by nhomar** [comment]
olvidalo

ya...
jejejeej

File: crm_claim_type/README.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
+CRM Claim Types
+==============
```
Comment (zaoral) [inline crm_claim_type/README.md:2]: This title line must be the same length as the title in line 1.

File: crm_claim_type/__openerp__.py (hunk 10 above)
Comment (osvalr) [inline crm_claim_type/__openerp__.py:28]: interesting, i just saw this https://github.com/Vauxoo/odoo/blob/8.0/openerp/modules/module.py#L237

**Pull request merged**